### PR TITLE
fix(spacing): fix spacing issues around the site

### PR DESCRIPTION
**Pull request by tw15egan**
Closes https://github.com/carbon-design-system/carbon-website/issues/1258
Closes https://github.com/carbon-design-system/carbon-website/issues/1259
Closes https://github.com/carbon-design-system/carbon-website/issues/1067

- Fixed spacing on homepage tiles
- Fixed spacing on top anchor links 
- Fixed spacing images, aligned the rows to the top 

- Ensured all `h2`-`h4` are `display: inline-block;` so they properly respect margins placed on them

### CODE REVIEW

**Comment by jeanservaas** [comment]
Much better, real quick:

- the distance between the intro paragraph and the anchor links needs to be 32px (not 24px) sorry, my bad, looks like it's still 24px

- Also, distance between anchor links and the first H2 (Carbon and accessibility) still looks too large by about 8px... can we just double check that it's 80px? 

- We miss you! But we know that you're following your heart by moving to Humbolt county to trim weed.

image

File: src/components/FeatureTile/feature-tile.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ a.feature-tile {
   @include breakpoint('md') {
     width: 50%;
     position: absolute;
-    right: 0;
+    right: 1rem;
```
Comment (alisonjoseph) [inline src/components/FeatureTile/feature-tile.scss]: Hmm, your fix for #1258 here might conflict with my fix for #1252 here https://github.com/carbon-design-system/carbon-website/pull/1265/files#diff-63c095e1b95ae5b451ab9537ec133b54R35

I added the `ibm-col-bleed` class, but it seems @jeanservaas and @designertyler suggested two different ways to fix the misalignment. 

@jeanservaas @designertyler can you confirm if the image should bleed into the column, or if the card should be moved inside the grid.

Comment (alisonjoseph) [reply src/components/FeatureTile/feature-tile.scss]: I think the bleed is correct, based on how the other tiles are displaying, but I could be wrong. 
screen shot 2019-03-05 at 10 35 26 am

Comment (alisonjoseph) [reply src/components/FeatureTile/feature-tile.scss]: Or with the tile just moved in to line up inside the grid.
screen shot 2019-03-05 at 10 36 17 am

**Comment by jeanservaas** [comment]
Hey guys, sorry I left this comment in #1259

This margin bottom distance is 24px now and i just need to change it to 32px! Thanks!

image

File: src/components/FeatureTile/feature-tile.scss (hunk 1 above)
Comment (alisonjoseph) [reply src/components/FeatureTile/feature-tile.scss]: confirmed with @jeanservaas that your fix is correct 👍

**Comment by alisonjoseph** [comment]
merging this in. will add spacing fix in another PR

**Comment by shixiedesign** [comment]
@alisonjoseph Sorry I'm just looking after this for Jeanie so hopefully I'm not misunderstanding the goal here. Jeanie madea  comment that this spacing below should be 32px, or 2rem. It's currently 1rem it seems. 
image

**Comment by alisonjoseph** [comment]
@tw15egan looks like one little update then we can merge this in

**Pull request merged**